pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -120,6 +120,37 @@ var osMap = map[string]string{
 	"windows_11":           "win11",
 }
 
+// Map of ovirt guest ids to instance type preference names.
+var osInstanceTypeMap = map[string]string{
+	"rhel_7_ppc64":       "rhel.7",
+	"rhel_7_s390x":       "rhel.7",
+	"rhel_7x64":          "rhel.7",
+	"rhel_8x64":          "rhel.8",
+	"rhel_9x64":          "rhel.9",
+	"ubuntu_12_04":       "ubuntu",
+	"ubuntu_12_10":       "ubuntu",
+	"ubuntu_13_04":       "ubuntu",
+	"ubuntu_13_10":       "ubuntu",
+	"ubuntu_14_04_ppc64": "ubuntu",
+	"ubuntu_14_04":       "ubuntu",
+	"ubuntu_16_04_s390x": "ubuntu",
+	"windows_10":         "windows.10", // windows also have virtio: e.g windows.10.virtio might be good to imports from ovirt?
+	"windows_10x64":      "windows.10",
+	"windows_2003":       "windows.10",
+	"windows_2003x64":    "windows.10",
```

Comment:
Will likely not work. Windows 10 will use EFI starting with the next release of common-instancetypes.